test/infra/infra-mariadb10/.env

```
@@ -11,3 +11,6 @@ MARIADB1_PORT=${PREFIX}306
 MARIADB2_PORT=${PREFIX}307
 MARIADB3_PORT=${PREFIX}308
 
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)